google_api_lib/task.py

```diff
@@ -15,16 +15,42 @@
 
 def create_google_sheets_helper(self, name):
     req_body = {"name": name}
-    response = (
+    # copy template sheet
+    file = (
         self._drive_service.files()
         .copy(
             fileId=settings.GOOGLE_SHEETS_TEMPLATE_FILE_ID,
             body=req_body,
+            fields="id,webViewLink",
+        )
+        .execute()
+    )
+
+    # transfer new sheet ownership back to OG owner, so that scripts can run
+    transfer_permission = {
+        "type": "user",
+        "role": "owner",
+        "emailAddress": self._sheets_owner,
+    }
+    self._drive_service.permissions().create(
+        fileId=file["id"],
+        body=transfer_permission,
+        transferOwnership=True,
+    ).execute()
+
+    return file
+
+
+def transfer_ownership(self, file_id):
```

Comment:
This looks unused? (I think the logic is already in create_google_sheets_helper)

Reply:
whoops thought I deleted that